lib/buildsystems/cmake.rb

```diff
@@ -12,7 +12,11 @@ def self.check?(bool = true)
   def self.build
     puts "Additional cmake_options being used: #{@cmake_options.nil? || @cmake_options.empty? ? '<no cmake_options>' : @cmake_options}".orange
     @crew_cmake_options = no_lto ? CREW_CMAKE_FNO_LTO_OPTIONS : CREW_CMAKE_OPTIONS
-    system "cmake -B builddir -G Ninja #{@crew_cmake_options} #{@cmake_options}"
+    if CREW_LINKER == 'mold'
+      system "mold -run cmake -B builddir -G Ninja #{@crew_cmake_options} #{@cmake_options}"
+    else
+      system "cmake -B builddir -G Ninja #{@crew_cmake_options} #{@cmake_options}"
+    end
```

Comment:
Simplify?
```
@linker = CREW_LINKER == 'mold' ? 'mold -run ' : ''
system "#{@linker}cmake -B builddir -G Ninja #{@crew_cmake_options} #{@cmake_options}"
```

Reply:
Logic simplified, but I changed the variable to be more descriptive.